src/tests/unit/tests/electron/platform/android/setup/steps/detect-permissions.test.ts

```diff
@@ -23,17 +23,15 @@ describe('Android setup step: detectPermissions', () => {
             .returns(_ => p)
             .verifiable(Times.once());
 
-        depsMock
-            .setup(m => m.stepTransition('configuring-port-forwarding'))
-            .verifiable(Times.once());
+        depsMock.setup(m => m.stepTransition('configuring-port-forwarding'));
 
         const step = detectPermissions(depsMock.object);
         await step.onEnter();
 
         depsMock.verifyAll();
     });
 
-    it('onEnter transitions to prompt-install-service as expected', async () => {
+    it('onEnter transitions to prompt-grant-permissions as expected', async () => {
```

Comment:
For symmetry with "on success", maybe "on failure"?
```suggestion
    it('onEnter transitions to prompt-grant-permissions on failure', async () => {
```